Xamarin.Forms.ControlGallery.MacOS/Xamarin.Forms.ControlGallery.MacOS.csproj

```diff
@@ -40,6 +40,7 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
+    <DebugType>full</DebugType>
```

Comment:
I think the value you're looking for is `portable`, and this condition could be moved to Directory.Build.props

Reply:
`<DebugType>full<DebugType>` is identical to `portable` on Mac, but not on Windows. `full` on Windows is a legacy pdb format.

`portable` means the same thing on both platforms (portable pdb format), so that should be used.